helm/net-exporter/templates/daemonset.yaml

```diff
@@ -59,6 +59,9 @@ spec:
         {{ if (.Values.NetExporter.Hosts) }}
         - "-hosts={{ .Values.NetExporter.Hosts }}"
         {{ end }}
+        {{ if (.Values.NetExporter.NTPServers) }}
+        - "-ntp-servers={{ .Values.NetExporter.NTPServers }}"
```

Comment:
Do we have a case in which we need to specify custom NTP servers? If not I'd drop that flag.

Reply:
Of course, we do. All the onprem installations are using custom NTP servers.